client/src/components/AuthMenu/AuthMenu.tsx

```diff
@@ -40,6 +43,11 @@ const AuthMenu = (): JSX.Element => {
         }}
         getContentAnchorEl={null}
       >
+        <MenuItem>
```

Comment:
```suggestion
        <MenuItem component={Link} to="/profile">
```

You can combine the MUI MenuItem with react-router's Link :)

Reply:
Thanks for these suggestions